modules/ROOT/pages/6.0-release-notes-plugins.adoc

```diff
@@ -0,0 +1,300 @@
+
+// Based on the code it looks like `media_scripts` was meant to be a way to provide an allowed list of <script>s that could be used to embed media. However, somewhere along the lines it broke and no longer works.
+
+As of the release of {productname} 6.0, this option has been removed completely.
+
+[[noneditable]]
+==== Noneditable
+
+The `noneditable` plugin is no longer a plugin. It is now a Core part of {productname} 6.0.
+
+In {productname} 6.0, there is no setting or specifying this functionality in the `plugins` option.
+
+All references to `noneditable` should be removed from your plugin configuration. Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load.
+
+[[paste]]
+==== Paste
+
+The `paste` plugin’s functionality is now a Core part of {productname} 6.0.
+
+In {productname} 6.0, there is no setting or specifying this functionality in the `plugins` option.
+
+All references to `paste` should be removed from your plugin configuration. Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load.
+
+The `paste_data_images` option now defaults to `true`. When `paste` was a plugin, this option was, by default, set to `false`, which prevented images being pasted from the local machine.
+
+The `mceInsertClipboardContent` argument, `content`, has been renamed. It is now `html`.
+
+The new name is a more accurate reflection of what sort of data the argument passes.
+
+NOTE: `PastePreProcess` also takes an argument, `content`. This argument, for this command, has not changed.
+
+
+[[print]]
+==== Print
+
+`print` functionality is no longer a plugin. It is now a Core part of {productname} 6.0.
+
+In {productname} 6.0, there is no setting or specifying this functionality in the `plugins` option.
+
+All references to `print` should be removed from your plugin configuration. Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load.
+
+[[table]]
+==== Table
+
+The `table` plugin’s functionality is, as of {productname} 6.0, entirely focussed on the user interface (UI) for creating and editing tables.
+
+The code that generates tables is now a Core part of {productname} 6.0.
+
+This, now core feature of {productname} 6.0, also includes the following fixes, changes, and improvements.
+
+* If the selected cells in the `tfoot` section of a table were header elements (ie were `th` elements), pressing the `table` row header toolbar button (which invokes the `mceTableRowType` command) returned the row as a header row.
+
+This was incorrect: the section takes precedence and cells in the `tfoot` section of a table should be declared as a footer row.
+
+In {productname} 6.0, selecting cells in the `tfoot` section of a table and pressing the row header toolbar button returns the row as a footer row, as expected.
+
```

Comment:
This reads weirdly to me, the way its rendering. I think it's all meant to be one bullet point?
```suggestion
* If the selected cells in the `tfoot` section of a table were header elements (ie were `th` elements), pressing the `table` row header toolbar button (which invokes the `mceTableRowType` command) returned the row as a header row.
+
This was incorrect: the section takes precedence and cells in the `tfoot` section of a table should be declared as a footer row.
+
In {productname} 6.0, selecting cells in the `tfoot` section of a table and pressing the row header toolbar button returns the row as a footer row, as expected.

```